test-app/backend/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "1.0.0",
   "description": "Backend example app.",
   "main": "lib/server.js",
+  "engines": {
+    "node": ">=14.14.0",
```

Comment:
I think the latest node is v18.7.0 (at least that's what mine got upgraded to). Should we use that here instead? Also, is there a way we can make these dynamic values (given that there are regular updates to node and npm?

Reply:
I'm using v14.18 lol